### ISSUE TITLE
Add CI integration and release builds

### ISSUE THREAD

**Issue by menski** [zeebe-io/zeebe-modeler#6]
- either use Zeebe Jenkins or Travis CI
- have a release process which creates a GitHub release with the release artifacts for the common platforms

__Tasks__

* [x] Setup nightly builds
* [x] Publish nightly to S3 bucket
* [x] Publish releases to GitHub releases

**Comment by nikku** [comment]
Merging https://github.com/zeebe-io/zeebe-modeler/pull/33 made the Zeebe Modeler adopt the new automatic release procedure implemented in the Camunda Modeler.

It works as follows:

* A developer creates a release issue to track the release progress (based on the [RELEASE_CHECKLIST](https://github.com/zeebe-io/zeebe-modeler/blob/master/docs/.project/RELEASE_CHECKLIST.md)
* She triggers the release via `npm run release`
* The release script will ask the developer for the version bump (minor, major, patch)
* The release will be tagged and pushed
* CI takes over and builds the released tag (including code signing) (cf. `.travis.yml` for the actual magic)
* CI publishes the release to https://github.com/zeebe-io/zeebe-modeler/releases
* The published artifacts are in `draft` state and must manually (after integration testing) be published

I'll update the [release procedures](https://github.com/zeebe-io/zeebe-modeler/blob/master/docs/.project/RELEASE_CHECKLIST.md) in a future PR.

---

__TLDR:__ @dominikhrn You may still trigger releases (if you'd like to do it). Just ensure you start it via the steps outlined above. We highly encourage you to create an issue containing the `RELEASE_CHECKLIST` to track the release progress. This allows you to offload work (such as integration testing on different platforms to us (@pinussilvestrus, @barmac, @philippfromme, @nikku).

**Comment by nikku** [comment]
The first nightly should build in the next 24h.

**Comment by nikku** [comment]
Closing issue, as all necessary improvements are in place.
